easybuild/easyblocks/n/netcdf4_python.py

```diff
@@ -70,11 +69,8 @@ def test_step(self):
 
     def sanity_check_step(self):
         """Custom sanity check for netcdf4-python"""
-        shlib_ext = get_shared_lib_ext()
-
         custom_paths = {
-            'files': ['bin/nc3tonc4', 'bin/nc4tonc3', 'bin/ncinfo'] +
-                     [os.path.join(self.pylibdir, x) for x in ['netCDF4.%s' % shlib_ext, 'netCDF4_utils.py', 'netcdftime.py']],
```

Comment:
OK, then this is good enough I guess, especially since it'll do the `import` check too.